src/util/votes.js

```diff
@@ -1,6 +1,21 @@
+import { contains } from 'ramda'
+
 export const readVotesFromStorage = () => localStorage.getItem('persistedVotes') ? JSON.parse(localStorage.getItem('persistedVotes')) : []
 
-export const writeVoteToStorage = songId => {
-  const persistedVotes = JSON.stringify([ ...readVotesFromStorage(), songId ])
-  localStorage.setItem('persistedVotes', persistedVotes)
+export const toggleVoteInStorage = songId => {
+  const isPreviousVote = contains(songId, readVotesFromStorage())
+  debugger
+  if (isPreviousVote) removeVoteFromStorage(songId)
+  else writeVoteToStorage(songId)
+}
+
+const writeVoteToStorage = songId => {
+  const persistedVotes = [ ...readVotesFromStorage(), songId ]
+  localStorage.setItem('persistedVotes', JSON.stringify(persistedVotes))
+}
+
+const removeVoteFromStorage = songId => {
+  const persistedVotes = readVotesFromStorage().filter(persistedSongId => persistedSongId !== songId)
+  debugger
```

Comment:
do we still need these `debugger` statements?

Reply:
i was just testing you